client/src/component/Rating.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+
+function Rating(props) {
+  const [vote, setVote] = useState(props.rating);
+
+  function handleClickUpVote() {
```

Comment:
I would make one function and pass an argument in the function on whether to add down or to add up. For example. 

```
const handleVote = (upVote) => 
{
  setVote( upVote ? vote + 1 : vote - 1)
}
...
<button onClick={() => handleVote(true)}>up vote</button>
<button onClick={() => handleVote(false)}>down vote</button>